onnxruntime/core/flatbuffers/flatbuffers_utils.h

```diff
@@ -42,6 +46,14 @@ onnxruntime::common::Status SaveValueInfoOrtFormat(
 
 void LoadStringFromOrtFormat(std::string& dst, const flatbuffers::String* fbs_string);
 
+// This macro is to be used on a protobuf message (protobug_msg), which will not create an empty string field (str_field)
+// if fbs_string is null
+#define LOAD_STR_FROM_ORT_FORMAT(protobug_msg, str_field, fbs_string) \
+  {                                                                   \
+    if (fbs_string)                                                   \
+      protobug_msg.set_##str_field(fbs_string->c_str());              \
+  }
```

Comment:
For load optional string, I have to use a macro,
I could use protobuf reflection if we are using full protobuf instead of protobuf-lite which does not support reflection